### ISSUE TITLE
Add options for feed update customization

### ISSUE THREAD

**Issue by webcatzz** [bcomnes/deploy-to-neocities#167]
The Neocities feed shows site updates using screenshots of the last four pages that were edited. Would it be possible to add an input to select which pages are deployed last? I like pushing my site with git but I lose the ability to customize how the update appears in the feed.

Thanks for the good work :-)

**Comment by bcomnes** [comment]
Is that available in the API? Open to adding it if so.

**Comment by web-catz** [comment]
Not in the official API...

But in async-neocities you could filter out certain files from filesToUpload in much the same way protected files are already filtered out from filesToDelete:

```js
// in neocitiesLocalDiff()
// given "delayedFiles: [list, of, paths]" in opts

const delayedFiles = new Set(opts.delayedFiles)

const filesToAdd = difference(difference(localFiles, ncFiles), delayedFiles)
const filesToDelete = difference(difference(ncFiles, localFiles), protectedFiles)
```

And then, in the deploy-to-neocities script, `client.upload()` them once all other files are uploaded. The only real trouble would be getting the list of paths to delay in the first place.

**Comment by bcomnes** [comment]
So the important files are the ones that upload last? 

What about being able to provide an array of strings that can be used in a sort predicate that are used to ensure they upload in that final order? Non-matching strings are just ignored. It would a static configuration most likely.

**Comment by bcomnes** [comment]
Wrote an update here: https://github.com/bcomnes/async-neocities/issues/80

tldr: I added basic sorting, but the larger vision for the API here is more work than I have interest in. Open to PRs if anyone wants to continue improving this. Closing for now.

**Comment by bcomnes** [comment]
I've been nerd sniped and thought about the issue a little more. 

I think I figured out the solution to the ordered chunk problem. 

So, when we have a sort algorithm, apply it to the entire set of files, chunk the array, and then reverse the upload chunks. 

This way, the first files in the sorted array are uploaded first, in the last chunk of the upload chunks. This should result in the desired effect of highlighting the first n files in the last upload chunk. 

In terms of the api for the cli and action, we can take an array of file names to upload first in the last chunk.
